### PR TITLE
Book repository

### PR DESCRIPTION
**Pull request by alisonreynoso**
## Summary
- Created BookRepository with method findByAuthor

## Trello Card
https://trello.com/c/pCFmyasI/15-crear-un-repositorio-de-books

### CODE REVIEW

File: src/main/java/wolox/training/repositories/BookRepository.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package wolox.training.repositories;
+
+import java.util.List;
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends Repository<Book, Long> {
+
+  List<Book> findByAuthor(String author);
```
Comment (julianvasquez-wolox) [inline src/main/java/wolox/training/repositories/BookRepository.java]: For this card, the method should return just one book. We can change the List<Book> for an Optional<Book> (so we can handle easier a null response) 😁

Comment (alisonreynoso) [reply src/main/java/wolox/training/repositories/BookRepository.java]: Got it! I've pushed the changes :)

**Review by julianvasquez-wolox — DISMISSED** [review]
Great! :D the naming is really accurate since the task was aimed to just one book 🥳

**Review by Jimenarosello — DISMISSED** [review]
Nice! 🚀

**Review by Jimenarosello — APPROVED** [review]
😄

**Review by julianvasquez-wolox — APPROVED** [review]

**Pull request merged**